extension/view/js/InPageTranslation.js

```diff
@@ -55,22 +150,31 @@ class InPageTranslation {
         this.startMutationObserver();
     }
 
+    addDebugStylesheet() {
+        const element = document.createElement("style");
+        document.head.appendChild(element);
+
+        const sheet = element.sheet;
+        sheet.insertRule("html[x-bergamot-debug] [x-bergamot-translated] { border: 2px solid red; }", 0);
+        sheet.insertRule("html[x-bergamot-debug] [x-bergamot-translated~=\"skipped\"] { border: 2px solid purple; }", 1);
+        sheet.insertRule("html[x-bergamot-debug] [x-bergamot-translated~=\"rejected\"] { border: 2px solid yellow; }", 2);
+        sheet.insertRule("html[x-bergamot-debug] [x-bergamot-translated=\"\"] { border: 2px solid blue; }", 3);
+        sheet.insertRule("html[x-bergamot-debug] [x-bergamot-translated=\"\"] [x-bergamot-translated~=\"is-excluded-node\"] { border: 4px dashed red; }", 4);
```

Comment:
What does `x-bergamot-debug` and `x-bergamot-translated` mean here? What is the purpose of this whole function? Is it usable in case of QE only?

Reply:
You can safely remove all getAttribute and setAttribute code. Its only there to help with identifying what blocks get send for translation, or why they get skipped. Nothing of this is related to QE.

The only functional usage of attributes is the `node.dataset.xBergamotId` and `data-x-bergamot-id` bits that are used to merge tags.

Reply:
Thanks for clarification @jelmervdl 👍🏾

Reply:
Btw, I couldn't find any reference to `data-x-bergamot-id` in the code.

Reply:
That's correct. `node.dataset.xBergamotId` [sets that attribute](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset) on the nodes when they are submitted to the wasm worker. Only during that time do the attributes exist. You should be able to see them at that moment in the DOM tree of the document as well as in the HTML string that's submitted to the WASM worker. I think they currently linger around after that, but in theory they can be removed again at the end of `merge()`.